src/Raven.Server/Documents/AI/ChatCompletionClient.cs

```diff
@@ -182,6 +188,52 @@ public async Task<AiResponse> CompleteAsync(JsonOperationContext context, HttpRe
 
         return (results.Result.ToString(), usage);
     }
+
+
+    private DynamicJsonArray CreateContentWithAttachments(string context, List<GenAiAttachment> attachments)
+    {
+        var content = new DynamicJsonArray
+        {
+            new DynamicJsonValue
+            {
+                ["type"] = "text",
+                ["text"] = context
+            }
+        };
+
+        foreach (var attachment in attachments)
```

Comment:
just checking if we are aware that for inline_data there is a 32MB limit for the whole request?